release/release_tests.yaml

```diff
@@ -403,7 +403,7 @@
 
   run:
     timeout: 2000
-    script: python tune_rllib/run_connect_tests.py
+    script: RAY_TRAIN_V2_ENABLED=1 python tune_rllib/run_connect_tests.py
```

Comment:
can we update this to use ray.tune.RunConfig?

https://github.com/ray-project/ray/blob/84e70db7649b8745044b46011219f55ae5e5d58e/release/ml_user_tests/tune_rllib/run_connect_tests.py#L58